core/src/main/scala/algebra/Order.scala

```diff
@@ -148,6 +163,27 @@ object Order extends OrderFunctions {
    */
   implicit def ordering[A](implicit ev: Order[A]): Ordering[A] =
     new Ordering[A] {
-      def compare(x: A, y: A) = ev.compare(x, y)
+      def compare(x: A, y: A): Int = ev.compare(x, y)
+    }
+
+  /**
+   * A `Monoid[Order[A]]` can be generated for all `A` with the following
+   * properties:
+   *
+   * `empty` returns a trivial `Order[A]` which considers all `A` instances to
+   * be equal.
+   *
+   * `combine(x: Order[A], y: Order[A])` creates an `Order[A]` that first
+   * orders by `x` and then (if two elements are equal) falls back to `y`.
+   *
+   * @see [[Order.andThen]]
+   */
+  implicit def orderMonoid[A]: Monoid[Order[A]] =
```

Comment:
we have generally shied away from creating default instances and leave those to standard.

How attached are you to this being implicit? We generally don't make implicits available in core, but in std. Also, can we imagine another monoid? For instance, trivially the order of left and right could be swapped (y whenEqual x). Are there others? For instance what if you take the first equal as a result, and on non-equal go to the next one (is that associative?)